common/src/main/java/org/conscrypt/ServerSessionContext.java

```diff
@@ -48,7 +45,7 @@ public ServerSessionContext() {
         NativeCrypto.SSL_CTX_set_session_id_context(sslCtxNativePointer, new byte[] { ' ' });
     }
 
-    public void setPersistentCache(SSLServerSessionCache persistentCache) {
+    void setPersistentCache(SSLServerSessionCache persistentCache) {
```

Comment:
This will need to be changed back to public for now for Android. (Feel free to mark it `@hide` or `@Internal`)